assets/javascript/developers.js

```diff
@@ -1,25 +1,43 @@
-function Developer(name, price, basePerformance, description, buffs, imageName) {
+function Developer(id, name, price, basePerformance, description, buffs) {
+  this.id = id;
   this.name = name;
-  this.price = price;
+  this.basePrice = price;
   this.basePerformance = basePerformance;
   this.description = description;
   this.buffs = buffs;
   this.level = 0;
   this.speed = 1;
-  this.imageName = imageName;
+
   this.performance = function() {
     let performance = basePerformance + Math.floor((this.level * basePerformance) / 2);
     let extra_performance = 0;
-    this.buffs.forEach(
-      function (buff) {
-        if(buff.active && buff.target === 'performance'){
+
+    this.buffs.forEach(buff => {
+        if(buff.active && buff.target === 'performance') {
           performance += performance * buff.value;
         }
       }
     );
-    performance += extra_performance;
-    return performance
-  }
+
+    return performance += extra_performance;
+  };
+
+  this.levelUp = function (amount) {
+    let level = this.level + amount;
```

Comment:
If you always increment just one, why are u using ` amount`
I think u can improve just 

```js
this.level++;
```

Reply:
Future proofing, many games allow you to increment in either 1, 10, 100 or max when you get to absurds amounts of money.